lib/Parse/ParseDecl.cpp

```diff
@@ -4606,6 +4606,14 @@ static void diagnoseOperatorFixityAttributes(Parser &P,
     if (fixityAttrs.empty()) {
       P.diagnose(OD->getOperatorLoc(), diag::operator_decl_no_fixity);
     }
+
+    for (auto it = Attrs.begin(); it != Attrs.end(); ++it) {
+      if (isFixityAttr(*it)) {
+        continue;
+      }
+      auto *attr = *it;
+      P.diagnose(attr->getLocation(), diag::operator_decl_should_not_contain_other_attributes, attr->getAttrName());
```

Comment:
Could you add a Fix-It to this diagnostic to remove the attribute (similar to line 4599).

---

Could you format the line to only contain 80 columns? I usually run `git-clang-format`, which (IIRC) you can install using Homebrew.